shared/js/media/crop_resize_rotate.js

```diff
@@ -255,7 +259,8 @@ function cropResizeRotate(blob, cropRegion, outputSize, outputType,
     }
 
     // Make sure the outputType is valid, if one was specified
-    if (outputType && outputType !== JPEG && outputType !== PNG) {
+    if (outputType && outputType !== JPEG &&
+        outputType !== PNG && outputType !== EXIF) {
       callback('unsupported outputType: ' + outputType);
       return;
```

Comment:
There is a problem a little bit below here at line 274.  It doesn't show in your diff so I can't comment on the line directly.  The problem is that if you pass an image/jpeg blob with outputType = 'image/jpeg+exif' and nothing at all needs to be changed, the code at line 274 will still think that the type is changing and so it will do lots of work and copy image instead of just returning it directly.

I recommend that after verifying the output value here, you add this code below:

```
var preserveEXIF = false;
if (outputType === EXIF) {
  preserveEXIF = true;
  outputType = JPEG;
}
```
Then in any of your code below that has a special case for outputType === EXIF you modify it to use this new preserveEXIF flag.